taxi/urls.py

```diff
@@ -16,7 +22,25 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
+    path(
+        "manufacturers/<int:pk>/update",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update",
+    ),
+    path(
+        "manufacturers/<int:pk>/delete",
```

Comment:
Ensure all URL paths end with a trailing slash to adhere to Django's conventions. Add a trailing slash here: `manufacturers/<int:pk>/delete/`.